metadata-search.ipynb

```diff
@@ -0,0 +1,1076 @@
+{
```

Comment:
<p>Unfortunately they are not, I will make a json file of these compounds for better visualization.</p><p> </p><p>Based on my search. I am assuming that the majority of perturbations are not categorized. The reason I have added the "NaN" is because there are some associated compounds that causes an "Unknown Injury / state"  </p>